.changeset/chilly-chicken-joke.md

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
I think we can remove this changeset. Doesn't seem like any package actually needs releasing from this.